plugins/client/graphql-kotlin-client-generator/src/main/kotlin/com/expediagroup/graphql/plugin/client/generator/types/generateTypeName.kt

```diff
@@ -51,8 +51,13 @@ import kotlinx.serialization.Serializable
 /**
  * Generate [TypeName] reference to a Kotlin class representation of an underlying GraphQL type.
  */
-internal fun generateTypeName(context: GraphQLClientGeneratorContext, graphQLType: Type<*>, selectionSet: SelectionSet? = null): TypeName {
-    val nullable = graphQLType !is NonNullType
+internal fun generateTypeName(
+    context: GraphQLClientGeneratorContext,
+    graphQLType: Type<*>,
+    selectionSet: SelectionSet? = null,
+    _nullable: Boolean = false
```

Comment:
```suggestion
    optional: Boolean = false
```

lets rename it to optional to avoid name clash and the _